.config/ansible-lint.spec

```diff
@@ -11,39 +11,21 @@ Version:        VERSION_PLACEHOLDER
 Release:        1%{?dist}
 Summary:        Ansible-lint checks ansible content for common mistakes
 
-License:        MIT
+License:        GPL-3.0-or-later AND MIT
```

Comment:
Drop MIT, keep GPL-3.0-or-later as that is the distribution licensing.

Reply:
This is the correct License tag per the new Fedora licensing guidelines. Both MIT and GPL-3.0-or-later apply to the distribution. Distributing the package under the GPL-3.0-or-later doesn't erase the MIT licensed code.